S/Spasm/build_tarballs.jl

```diff
@@ -0,0 +1,66 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spasm"
+version = v"1.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/laurentbartholdi/spasm.git", "072719a40c837e447dfe4ae9e4941c60d9a28eda"),
```

Comment:
Why a fork and not the upstream repo https://github.com/cbouilla/spasm?  As far as I can tell, commit https://github.com/cbouilla/spasm/commit/072719a40c837e447dfe4ae9e4941c60d9a28eda is present there.